mkdocs.yml

```diff
@@ -29,38 +30,38 @@ theme:
         logo: fontawesome/solid/house
         annotation: fontawesome/solid/circle-info
 
-# Navigation for the site 
+# Navigation for the site
 nav:
     - Home: index.md
-    - Over het algoritmekader: 
-        - overhetalgoritmekader/index.md
-        - overhetalgoritmekader/definities.md
-        - overhetalgoritmekader/CONTRIBUTING.md
+    - Over het algoritmekader:
+          - overhetalgoritmekader/index.md
```

Comment:
All these extra tabs are not really part of your code, The Algorithmkader may decide to ignore this because of scope creep